src/Gateways/Clients/SdkClient.php

```diff
@@ -237,6 +237,12 @@ protected function prepare_request_objects() {
 				$this->builder_args['requestMultiUseToken'] = array( true );
 			}
 		}
+		// Checks if order contains a subscription and requests a muti-use token.
+		if(function_exists("wcs_order_contains_subscription")){
+			if(isset($this->builder_args['orderId']) && wcs_order_contains_subscription(wc_get_order($this->builder_args['orderId'][0]))){
```

Comment:
The white spaces comment applies in here too, except for the ['orderId'] part, where it is fine as it is, according to the coding standards.